packages/wallets/src/AppContent.tsx

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
-import WalletList from './components/WalletList';
+import { useAuthorize } from '@deriv/api';
 import WalletsCarousel from './components/WalletCarousel';
-import IcBrandDerivGo from './public/ic-brand-derivgo.svg';
+import WalletList from './components/WalletList';
+// import IcBrandDerivGo from './public/ic-brand-derivgo.svg';
```

Comment:
Comment out the SVG usage for now until the configuration is added to `@deriv/core`.